packages/components/src/components/vertical-tab/vertical-tab-content-container.jsx

```diff
@@ -3,14 +3,34 @@ import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import Icon from '../icon/icon.jsx';
 
+const SideNotes = ({ side_notes }) => {
+    return (
+        <div className='dc-vertical-tab__content-side-note'>
+            {side_notes?.map((note, i) => (
+                <div className='dc-vertical-tab__content-side-note-wrapper' key={i}>
+                    {note}
+                </div>
+            ))}
+        </div>
+    );
+};
+
 const Content = ({ is_routed, items, selected }) => {
     const selected_item = items.find(item => item.label === selected.label);
     const TabContent = selected_item.value;
-
     const [side_notes, setSideNotes] = React.useState(null);
 
+    const note_array = [];
+
+    const setNotesSync = notes => {
+        note_array.unshift(notes);
+        setSideNotes(notes);
+    };
+
     React.useEffect(() => {
         setSideNotes(null);
+        setSideNotes(note_array[0]);
```

Comment:
Do we still need to the state update to `null`, could we do something like this:

```JS
setSideNotes(note_array[0] ?? null);
```